pkg/metrics/metrics.go

```diff
@@ -28,10 +28,10 @@ const (
 	LabelISBService         = "isbsvc"
 	LabelPipeline           = "pipeline"
 	LabelVertex             = "vertex"
-	LabelMonoVertex         = "mvtx"
 	LabelVertexReplicaIndex = "replica"
 	LabelVertexType         = "vertex_type"
 	LabelPartitionName      = "partition_name"
+	LabelMonoVertex         = "mvtx"
```

Comment:
```suggestion
	LabelMonoVertexName         = "mvtx_name"
```

I saw you tested the entire PR using a real mono vertex, I am curious why the test looks good with the label name not matching with rust code.

Reply:
In our case there is no filtering on labels. Hence, we are able to retrieve the metric vals